.scrutinizer.yml

```diff
@@ -0,0 +1,32 @@
+before_commands:
+    - "composer install --no-dev --prefer-source"
+
+tools:
+    external_code_coverage: true
+    php_code_sniffer:
+        enabled: true
+        config:
+            standard: PSR2
+        filter:
+            paths: ["src/*", "tests/*"]
+    php_cpd:
+        enabled: true
+        excluded_dirs: ["tests", "vendor"]
```

Comment:
I would include the tests as well, the tests should be written with the same care as the code that is written for production